### ISSUE TITLE
[CAIA Intake] Benefits and Claims Non-Disability Experience: Review Burial Form Updates

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#68984]
## About your team

- Team name: Benefits and Claims | Lifestage Benefits Crew | Non-Disability Experience
- OCTO-DE product owner: Emily Theis
- Product manager: Laura Steele
- Designer: Fiorella Glavez
- Accessibility specialist: none
- Content writer: none
- Slack channel: benefits-non-disability
 
### Describe what you need in a few sentences:
- We would like CAIA to review the critical updates we are making to the 530EZ burial form, so it can be successfully processed.
- We ask that the scope of CAIA's review be limited to the form pages we are touching, so we can quickly release a working MVP version of the updated form. Future work will focus on taking a more comprehensive look at the form to determine additional enhancements, replace depreciated design components, and address accessibility issues.
  - We intend to fix defect-0 and defect-1 level A11y errors, but we would like to limit those fixes to the pages we are touching with these field updates. 
- The current online form is missing critical fields that were added to the paper form over the last 2 years. These missing fields are causing significant processing delays and putting additional burden on Veterans.
- The changes include adding missing fields and removing depreciated fields.

- Fixing the 530 form is a high-priority for OCTO and the VBA.

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [x] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [x] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [x] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] #91543
- [ ] Something else 
#### IA
- [x] User Flows
- [ ] Page placement
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [x] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next step is** collaborating with VBA SMEs to answer a few outstanding questions and updating the 530 form after CAIA sign-off.
- **Provide date if available**: ASAP

#### Timing for your next step: 
  - [x] Next week
  - [x] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [x] No

### If you're conducting research, when is that starting? 
**Provide date if available:** xx/xx/xxxx
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [ ] Yes
  - [x] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users on:** December 2023 - January 2024
  
### Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- **Ticket (number)**: #65251 
- **Collab. Cycle Ticket Opened (date)**: 9/08/2023
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [x] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [x] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [x] Accessibility
- [x] Content
- [x] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [x] Accessibility
- [x] Content
- [x] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Product outline**: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/pension/2019%20files/README.md
- **User flows**: https://sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7
- **Prototype or mockups**: https://sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7
- **If this is in Sketch, which version or specific page should we focus on?** Within each section, provide feedback on the "proposed" version of the screens.
- **Content source of truth**: Within Sketch file.
- **Research plan:** DNA
- **Any epics/issues that may be helpful**: [Update Burial 530 Online Form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63351)
- **Link to testing accounts** (please do not paste them in this ticket): https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/burials-memorials/burial-allowance/530-staging-test-accounts.md

### Have you already worked with someone from OCTO leadership?
- [x] Emily Theise, Julie Strothman
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.
- [x] Schedule a kick-off call with your team and CAIA for your project **(Recommended)**

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next. 

```[tasklist]
### CAIA Ops Tasks
- [x] Scheduled Kickoff Meeting - Mon Nov 6th - 4:30 PM ET
- [x] https://github.com/department-of-veterans-affairs/va.gov-team/issues/69301
- [x] Recording Link from Kickoff [Needs Added to Deliverable](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/meeting-notes/2022-11-06-Kickoff-Meeting-Burials-Form-Updates-Non-Disability-Experience-Team.md) by @coforma-terry
```

```[tasklist]
### CAIA a11y Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/69302
- [ ] [a11y Support] Benefits and Claims Non-Disability Experience: Burial Form Updates: Midpoint Prep `Midpoint Date TBD - Convert Task to Ticket Pending`
- [ ] [a11y Support] Benefits and Claims Non-Disability Experience: Burial Form Updates: Staging Prep `Staging Date TBD - Convert Task to Ticket Pending`
- [ ] [a11y attend meeting] Benefits and Claims Non-Disability Experience: Burial Form Updates: Watch or Attend Midpoint `Date TBD`
- [ ] [a11y attend meeting] Benefits and Claims Non-Disability Experience: Burial Form Updates: Watch or Attend Staging `Date TBD`
- [ ] [a11y Support] Communicate Midpoint Prep Finidngs to the VFS Team via Slack and Ticket tags
```

```[tasklist]
### CAIA Content Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/73370
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/74092
```

```[tasklist]
### CAIA IA Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/78479
```

**Comment by sara-amanda** [comment]
## CAIA Update
- [x] **11/6/2023** [Kickoff Meeting Deliverable](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/meeting-notes/2022-11-06-Kickoff-Meeting-Burials-Form-Updates-Non-Disability-Experience-Team.md)
- [ ] **Recording Link:** @laurwill or @coforma-terry to add when available

**Comment by steele-lm** [comment]
@sara-amanda @coforma-terry I added the recording link [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/meeting-notes/2022-11-06-Kickoff-Meeting-Burials-Form-Updates-Non-Disability-Experience-Team.md)

**Comment by aliyahblackmore** [comment]
Hi @steele-lm - in our kickoff, meeting you all mentioned that there's still SME feedback that you all are waiting on across a few screens. Have you all received that feedback and adjusted the designs to reflect any needed changes?

Thanks so much!

**Comment by steele-lm** [comment]
@aliyahblackmore We have a meeting scheduled with our burial SMEs on Thursday (11/16). If you have any specific questions, please reach out to @juliepedtke and myself!

**Comment by aliyahblackmore** [comment]
@steele-lm thanks for letting me know!  I'll tag you both in sketch questions.

**Comment by aliyahblackmore** [comment]
Hi @juliepedtk - I'm just flagging that I've started to add comments and questions to the form intro and personal information screens in the [Sketch link provided by you all](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/61980607-BE54-4442-BC1A-DD3595F695FF/canvas). As we chatted in our kickoff, comments are on the "Proposed" screens.

Additionally, accessibility and IA finished their review of **all** of the screens in the form (their teams should have shared feedback with you all - if not, let me know). I'll continue adding my feedback/questions tomorrow and will make recommendations to the content with the purple annotations (in scope for the content review), while also calling attention to any other items flagged by IA and accessibility that impact the content outside of the purple annotations.

cc: @strelichl @laurwill

**Comment by jennymayoco** [comment]
Hi @juliepedtke @steele-lm from the IA side, we recommend updating the base URL so that the form is up to date with standards. The standard is a URL with 2-3 words followed by form number, such as `form-name-form-####`. Also, the breadcrumbs are currently `Home > Burials and memorials > Burial allowance > Review burial benefits application` but the URL is `www.va.gov/burials-and-memorials/application/530/` which is missing the burial allowance page. Updating the URL will ensure it matches the navigation path as well. Would this recommendation fall within scope for the MVP?

**Comment by steele-lm** [comment]
@jennymayoco This is a great call out! I think it would most likely fall under a "fast follow" activity given that the first priority is reducing delays in benefits due to missing form fields.

**Comment by jennymayoco** [comment]
@steele-lm that makes sense! We don't have any additional IA recommendations for the MVP at this moment and understand that further IA work might be until later versions.

cc @aliyahblackmore @SarahKay8

**Comment by aliyahblackmore** [comment]
Hi @steele-lm and @juliepedtke ! 

**Just tracking some updates here:**

- Our content feedback and outstanding question are in the [sketch screens](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/61980607-BE54-4442-BC1A-DD3595F695FF/canvas). When you all reply to our questions, we need to make adjustments to content before MVP launch.  
- When you all update the designs based on the SME feedback that you all got from your meeting last Thursday, let us know and we can make any needed adjustments to the content (the areas with the pink notes).
- I'll be OOO starting tomorrow (11/21) until Friday (11/24). If there are any urgent questions tomorrow or early on Wednesday, you all can tag Laura here. 

cc @laurwill

**Comment by aliyahblackmore** [comment]
Hi @fiorella-io and @juliepedtke - 

I'm seeing that you all are replying to some of my questions in sketch today! 
When you have all of the answers posted there, could you just let us know in this thread so that we can turn back to making any updated content recommendations? 

Thanks for your continued collaboration!

**Comment by aliyahblackmore** [comment]
HI @fiorella-io 

Thanks for letting me know. I'll look out for a ping from you when you all get the answers from SMEs and I'll make updated content recommendations.

Also, it looks like you all are updating Sketch files right now, so I can't reply to your [comment ](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/mPM4JZg)on the sketch screen about the "Note" alert content. This is my reply and I'll post it there when Sketch is done updating.

**Reply:**

This isn't outlined in purple (to review ahead of MVP launch) so this wasn't part of our initial review, but this "Note:" content is fine as is.

**Comment by aliyahblackmore** [comment]
Hi @fiorella-io ! 

I also noticed that there are some other content pieces either on new screens (i.e. [this in progress screen](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/DEVjleJ#Comment)) or with purple outlines that you mentioned you didn't include before. Can you just confirm that the designs that include purple reflect what we need to review and that there aren't additional areas that you'll plan to outline in purple?

**Comment by fiorella-io** [comment]
Hi @aliyahblackmore, can you please review the pages on [Step 5](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/E1B6C12E-B2F8-4A50-9CC1-1F44853FA3C6/canvas), and the [Review expanded page](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/MmW5eQa) to make sure we've captured all field label changes. Those are the only pages we need that I had not labeled with purple before.

**Comment by aliyahblackmore** [comment]
**Cross posting here** 

Julie/Fiorella shared the invite to the VBA stakeholder call (this afternoon) a few minutes ago.

I won't be able to join, but they have the outlined questions above and Fiorella flagged that they'll share some of the lower impact questions via Teams. I asked them to include me on the Teams messages.

**Comment by fiorella-io** [comment]
Hi @aliyahblackmore! Here are my responses. 

### Intro page screen

* _TBD. I asked this question on Teams. I’ll add it here once our SMEs get back to us on Monday._ 

### Military history screen
* Not without DD214 or other separation papers
* We didn’t touch base on the respondent burden, but we will increase it from 15mins to 30 minutes to match the paper form
* Yes, if someone selects that they have their DD214, they will be able to skip the service history. However, we learned from our SMEs that a Veteran can serve, receive a DD214, and then serve again, so they would have more than one. We need to add that if they are uploading a DD214 make sure it includes all their service periods. We should include this copy in Step 3 where we ask if they have a DD214, and in Step 5, where we ask to upload it. 

### Previous names screen

* Shouldn’t matter if it’s service-connected. If they served under a different name, they want to know that for service and non-service connected and they want to know what services were rendered under that name. They did mention that this information can be on a DD214.   Also, since it’s a question on the paper form, they want to make sure it’s also on the new burial form. 

### Another name button

* Having served under different names is an edge case, but the existing burial online experience allows for multiple entries. We want to keep this functionality on the new burial form

### Burial/plot/transportation allowance screens
* Yes the claimant needs to be responsible for the expenses
* My example was not a good one 😅
* If they answer that they are NOT responsible for the expenses (we ask this responsibility question for plot and burial expenses), they will not receive an allowance for those two benefits. Nothing else will appear or change in the form if No is selected

### Plot or internment allowance screens
* _TBD on teams_
* _TBD on teams_

### Upload screen for non-service-connected death allowance
* If the claimant has previously applied for benefits VA might already have the Veteran’s medical records. If they haven’t previously applied, they would get a letter asking for medical records after they submit the burial form.
* The death certificate would include this information. They mentioned that the eligibility and evidence table on the paper has the latest content and that the How to Apply page might not.

### Upload screen service-connected death
* Just by selecting “Yes” that they are responsible for the expenses, if it’s service-connected, that’s all the evidence they need.   
 
They also mentioned that a claimant doesn’t need to submit evidence for plot or burial allowance. Only if they are claiming transportation allowance they would need an itemized receipt. They mention that most of the rejection is failure to send an itemized receipt, so we’ll need to emphasize this on the Transportation upload page. We will delete the non-service-connected burial, serviced-connected burial, and plot allowances pages since they don’t need to submit evidence for those.

Also since you are now on the Team chat with our SMEs feel free to ask them any additional questions you may have. Thanks!

**Comment by aliyahblackmore** [comment]
@fiorella-io Thanks so much, Fiorella!!

I was OOO on Friday, so I'll catch up on these replies today and I'll take a look at any additional replies on Teams and ask questions there directly.

**Comment by aliyahblackmore** [comment]
Hi @fiorella-io - following up here!

I checked my Teams chat and VA email and I'm not seeing a chat or thread with SMEs. Could you loop me in?

**Comment by fiorella-io** [comment]
Hi @aliyahblackmore, I added the responses to the "Intro page screen" and "Plot or internment allowance screens
" sections.

**Comment by aliyahblackmore** [comment]
Thanks @fiorella-io ! And as we chatted on Slack, I'm now on the Teams thread so I'll send any follow up SME questions there.

**Comment by fiorella-io** [comment]
Hi @aliyahblackmore let me know if there are any additional content changes. We need to schedule our midpoint review, and they prefer all designs to have the final copy. 

As I mentioned before, we have made some changes to Step 5, the document upload page, since transportation is the only benefit that needs evidence, we have removed the non-service-connected burial, serviced-connected burial, and plot allowances pages

**Comment by aliyahblackmore** [comment]
Hi @fiorella-io Laura and I are meeting tomorrow to review the content changes - I'll share them with you all in Sketch after.

**Comment by fiorella-io** [comment]
Perfect! All the changes we have made are in the V2 section of our Sketch file. We don't have a v2 of Step 5, since you had additional questions regarding evidence. I've highlighted the sections that you can review. 
<img width="274" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/91498500/a051bfe2-2401-4f16-8d73-97ef8633c8c2">

**Comment by aliyahblackmore** [comment]
@fiorella-io great - I'll continue to add sketch comments to the pieces of content that still need feedback in the MVP/V1 section.  Also, I noticed that you all now have a  "V1" label instead of the previous "MVP" label. 

Are you all still launching the MVP version?

**Comment by aliyahblackmore** [comment]
Thanks @fiorella-io. 

The section label that was previously "MVP" is now "V1" - are you all no longer launching the MVP version?

**Comment by fiorella-io** [comment]
Yes, we are still launching an MVP. This whole file is the MVP

**Comment by aliyahblackmore** [comment]
Hi @fiorella-io! 

**Notes:**

- All of my updates to the content based on your notes and SME replies are in the V2 sketch screens and the V1 sketch screen for Step 5.  

- As we chatted, to remain in scope, I reviewed the content with the purple annotations. But I also included a few other recommendations to better align content across screens with VA style standards and to accurately frame the question around burial expenses. 

- I also noticed that there was a new piece of content on the [benefit selection screen](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/VEY5EJw). I made recommendations there.

**A few last things:** 

- There’s one error message that you still need to confirm with engineers. When you have that info, I can adjust that content. It’s on [this upload screen](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/4oxKk7d).

- There's a question on the DD214 or other separation documents upload screen. I adjusted the content to include information about "all service periods," based on your above note, but I noticed in the flow that there's no an additional upload button. You all will need to confirm with SMEs that the content recommendation is an accurate interpretation and I can make any adjustments, if needed.

- And I pinged you again on the [other evidence screen](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/AZVYgmM) – based on the previous upload screens I’m not sure that this additional screen is necessary. I included content recommendations just in case.


I need to sign off for this evening, but I'll check in on this ticket next week.

Thanks for the continued collaboration.

**Comment by fiorella-io** [comment]
Thank you so much @aliyahblackmore! I'll review everything and get back to you.

**Comment by aliyahblackmore** [comment]
Hi @fiorella-io! Happy New Year! 

I pinged you on a few comments in [Sketch (V2)](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/4D9CC817-5BE7-46F7-A3F8-7B1FB7A54561/canvas#Comment). It seems like there are some new adjustments to the flow and content on the military service (branch of service drop down and previous name "services rendered" changes) and upload screens (transportation costs and DD214 upload screen are not longer there). 

Are all of the recent flow/content updates ready? If so, I'll take a look at the content and make any needed adjustments.

**Comment by fiorella-io** [comment]
Hi @aliyahblackmore Happy new year! 🎉
We've been updating some pages based on the midpoint review feedback we got.

The DD214 upload now resides right after the DD214 question. The transportation upload page has stayed on Step 5.
Feel free to take another look at everything. We just finished implementing all the MUST and SHOULD feedback for our MVP.

**Comment by aliyahblackmore** [comment]
Thanks @fiorella-io I'll take a look tomorrow!

Could you link the MUST and SHOULD feedback here?

**Comment by aliyahblackmore** [comment]
**Updates here:** 

- Their team shared that they'll need a content review of a few modal screens
    [Step 3: Service periods](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/v/74MwL2/a/JAKwVpM)
    [Step 3: Veteran’s previous name](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/vRMPAv1)
    [Step 5: Upload documents](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/v/mknZQL/a/GoxVRpY/)
- They made a slight adjustment to the flow -  I'll need review the military history screens again and update content.
- They removed the "2 year" information from the form and we'll make updates to the "filing time limit" section of the page. Tracking those content needs in https://github.com/department-of-veterans-affairs/va.gov-team/issues/73370.

**Comment by aliyahblackmore** [comment]
**Tracking some updates here:**

- Just tracking here that I did a review of all the [Figma screens](https://www.figma.com/file/jpJQSCKiGegacdW4RH5uQB/Burial-530-Redesign-MVP?type=design&node-id=148-30926&mode=design&t=jPqGBHBTPxSwBCHY-0) to confirm any content changed are in Figma. 
- I noticed there's a new  screen in Figma - the confirmation screen. I'll wait for Fiorella to confirm whether they'll need a review for MVP.
- Laura S. shared (2/8) that they have a plan for the launch. She shared details via Slack - I'll take a look next week.

**Comment by aliyahblackmore** [comment]
**Cross posting and tracking some updates here:** 

- 2/13 we walked through the release plan and flows for auth/unauth users for the V1 and V2 versions of the form.
- The product team will confirm a few questions about showing different confirmation screens. A different screen would need to show for any in-progress V1 forms -- content needs to explain that VBA will contact them with more questions about their application.


**Some content next steps:** 

- I'll reach out to SMEs about how they request additional information for an users that submit V1 of the form. Will they send them a PDF version of the form to fill out? Will they always send a PDF or will they sometimes just request specific documents? 
- We'll make updates to the How to apply static page to align with the new form updates (including an alert about the form changes - similar to the pension form).  I'll have SMEs review and approve the accuracy of those content changes and I'll share a SharePoint document with you all that includes the content for the incremental launch -- content that users who can use the new form will see
- And we'll also share with SMEs that we'll include an alert on the static page and (TBD) the form intro page, similar to the pension form. Alert will let users know that there are updates to the form and they'll need to review their answers and answer new question.
- Closer to the launch date, maybe in early March, let's set up a meeting with the copy editor to discuss the incremental launch and the review she'll need to do once the content is in staging. You all will have to turn on the flipper and she'll take a look at the content. I'll set a calendar reminder to circle back on this meeting and finding a time in early March!

**Comment by aliyahblackmore** [comment]
**Tracking some updates here:** 

- I'll plan to turn back to this work next week (I've have to work on a very high priority content item for another team this week).
- Product team is expecting some additional SME feedback next week. 
- I'll send a question via Teams about what happens after forms are submitted. For example, does VBA review the application and send a letter in the mail with the decision.
- Still waiting on confirmation from product team about the release plan for saved and in-progress forms. I flagged the content and UX considerations for the team last week. They'll circle back.

**Comment by jennymayoco** [comment]
Hi @steele-lm @fiorella-io I created an [IA spec](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/burial-allowance-form-updates.md) to document the entry point for the [about form page ](https://www.va.gov/find-forms/about-form-21p-530ez/). We're also updating the best bet so it links to the online application instead of the ["how to apply" page](https://www.va.gov/burials-memorials/veterans-burial-allowance/). Other than that, we don't have additional IA feedback.

In addition, we had previously suggested updating the form link so that it's up to date and follows URL standards and that this work would begin in a later version. This form will eventually be moving to the family benefits hub and so as not to duplicate the work, we can plan to update the URL then. Looping in @mnorthuis in case you have any other questions about the family benefits hub.

**Comment by aliyahblackmore** [comment]
**Tracking a general notes here for awareness (this was already communicated via Slack and [Figma](https://www.figma.com/design/jpJQSCKiGegacdW4RH5uQB?node-id=1796-29743#788636932)):** 

- I made initial content adjustments (4/30) to the [updated FDC flow designs ](https://www.figma.com/design/jpJQSCKiGegacdW4RH5uQB/Burial-530-2024-Updates-MVP?node-id=1799-37607&t=1DIsNZZBTmFmUFBG-0)that the product team decided to move forward with. I'm in the process of making additional adjustments based on new information from SMEs.
- On 5/6 and 5/7, SMEs shared via Teams that medical records or authorization for VA to retrieve medical information is only an optional document if the applicant is claiming a burial allowance for a service-connected death. 
- Given this new information for SMEs, the form flow should be updated with conditional logic to decrease cognitive load and increase clarity. Currently, all users see information about medical records, but the information is only necessary for users who select "service-connected burial allowance." Because the product team previously decided to leave the flow as it is, they team should decide when and how they will update the flow based on the new SME information from this week.
  - **One suggestion** - if someone selects that they're claiming a burial allowance for a service-connected death, later in the flow, the supporting documents are shown conditionally, and they're shown a upload screen for the medical record/authorizing for release of non-VA medical information documents. On the confirmation screen, details about medical records are shown conditionally. The designers and product team should determine how to integrate some sort of conditional logic in to the form flow and determine when this can happen.
    - And because there aren't currently specific questions about medical records on the PDF version of the form, the product team needs to confirm how any additional flow updates would impact the back end. 


**FYSA** @aprocik1 @fiorella-io and @steele-lm

**Comment by steele-lm** [comment]
@strelichl I confirmed with @fiorella-io that the outstanding feedback from the most recent comment (FDC and medical records) have been incorporated into the burial form, so I believe this ticket can be closed as there are no outstanding CAIA items.

**Comment by mnorthuis** [comment]
Content and IA tasks for this effort is complete.  Removed sub-issue related to possible future content updates given that can be done independently of this work.
